isis/src/mro/apps/pds2hideal/MroHiriseIdealPdsImportLabel.trn

```diff
@@ -3,7 +3,7 @@
 Group = SpacecraftName
   Auto
   InputKey       = SPACECRAFT_NAME
-  InputPostion   = ROOT
+  InputPosition   = ROOT
```

Comment:
wow, hard to believe that's been in there all this time

Reply:
It wasn't being used. There was a version of this file already in the directory, but it wasn't being used, so there were no functional issues with this file.